Sources/MuxPlayerSwift/PublicAPI/Options/PlaybackOptions.swift

```diff
@@ -24,7 +24,42 @@ public enum ResolutionTier {
     case upTo2160p
 }
 
-extension ResolutionTier {
+/// The min resolution tier you'd like your asset to be streamed at
+public enum MinResolutionTier {
+    /// By default no resolution tier is specified and Mux
+    /// selects the optimal resolution and bitrate based on
+    /// network and player conditions.
+    case `default`
+    /// The asset will stream with a resolution that does
+    /// not exceed 480p (640 x 480)
+    case upTo480p
+    /// The asset will stream with a resolution that does
+    /// not exceed 540p (960 x 540)
+    case upTo540p
+    /// The asset will stream with a resolution that does
+    /// not exceed 7200p (1080 x 720)
+    case upTo720p
+    /// The asset will stream with a resolution that does
+    /// not exceed 1080p (1920 x 1080)
+    case upTo1080p
+    /// The asset will stream with a resolution that does
+    /// not exceed 2440p (2160 x 4096)
+    case upTo1440p
+    /// The asset will stream with a resolution that does
+    /// not exceed 2160 p(2560 x 1440)
+    case upTo2160p
+}
+
+public enum RenditionOrder {
+    /// By default no rendition order is specified
+    case `default`
+    /// The asset will choose renditions in ascending order
+    case asc
```

Comment:
These should be fully spelled out, see [these Swift API design guidelines](https://www.swift.org/documentation/api-design-guidelines/#use-terminology-well).

Reply:
I am bit stuck here, @andrewjl-mux  can you help with the comment content ?